source/renderer/app/components/wallet/settings/UndelegateWalletButton.scss

```diff
@@ -0,0 +1,9 @@
+.root {
+  color: var(--theme-color-error) !important;
+  font-weight: 500 !important;
+  height: 36px !important;
+  line-height: 36px !important;
+  margin-left: 20px;
+  min-width: 160px !important;
+  width: 160px !important;
+}
```

Comment:
Same as above. Use wrapper + `:global` instead of !important

Reply:
@tomothespian Good catch. I didn't make this one new in this PR. Just copied from other existing style, and didn't try to beautify it. Will fix now.